### ISSUE TITLE
New `get_alias` tests failing on 6.x branch

### ISSUE THREAD

**Issue by ywelsch** [elastic/elasticsearch#30806]
New REST tests for `get_alias` have been added in #29513
They fail in mixed-cluster tests on the 6.x, see for example https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-unix-compatibility/os=centos/1076/console
I've reverted the commit on the 6.x branch (see ad432315cba)

Log output of test failures

```
"TEST_FAILURE",
 {
  "description": "ID#test {p0=indices.get_alias/10_basic/Get aliases via /_alias/_all}(org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT)",
  "message": "Failure at [indices.get_alias/10_basic:51]: field [test_index_3] has a true value but it shouldn't\nExpected: (\"\" or equalToIgnoringCase(\"false\") or \"0\")\n     but: was \"{aliases={}}\"",
  "trace": "java.lang.AssertionError: Failure at [indices.get_alias/10_basic:51]: field [test_index_3] has a true value but it shouldn't\nExpected: (\"\" or equalToIgnoringCase(\"false\") or \"0\")\n     but: was \"{aliases={}}\"\n\tat __randomizedtesting.SeedInfo.seed([5BD30D4D76C681C2:D3873297D83AEC3A]:0)\n\tat org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:356)\n\tat org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:333)\n\tat jdk.internal.reflect.GeneratedMethodAccessor15.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1713)\n\tat com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:907)\n\tat com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:943)\n\tat com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:957)\n\tat com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)\n\tat org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)\n\tat org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)\n\tat org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)\n\tat org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)\n\tat org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)\n\tat com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)\n\tat com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)\n\tat com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:817)\n\tat com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:468)\n\tat com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:916)\n\tat com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:802)\n\tat com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:852)\n\tat com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:863)\n\tat org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)\n\tat com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)\n\tat org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)\n\tat com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)\n\tat com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)\n\tat com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)\n\tat com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)\n\tat org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)\n\tat org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)\n\tat org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)\n\tat org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)\n\tat com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)\n\tat com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)\n\tat java.base/java.lang.Thread.run(Thread.java:844)\nCaused by: java.lang.AssertionError: field [test_index_3] has a true value but it shouldn't\nExpected: (\"\" or equalToIgnoringCase(\"false\") or \"0\")\n     but: was \"{aliases={}}\"\n\tat org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)\n\tat org.junit.Assert.assertThat(Assert.java:956)\n\tat org.elasticsearch.test.rest.yaml.section.IsFalseAssertion.doAssert(IsFalseAssertion.java:59)\n\tat org.elasticsearch.test.rest.yaml.section.Assertion.execute(Assertion.java:76)\n\tat org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:349)\n\t... 36 more\n",
  "throwableString": "java.lang.AssertionError: Failure at [indices.get_alias/10_basic:51]: field [test_index_3] has a true value but it shouldn't\nExpected: (\"\" or equalToIgnoringCase(\"false\") or \"0\")\n     but: was \"{aliases={}}\"",
  "throwableClass": "java.lang.AssertionError",
  "assertionViolation": true,
  "assumptionViolation": false
 }
]
```

**Comment by jasontedor** [comment]
Closed by #30814

**Comment by javanna** [comment]
thanks @jasontedor for closing
